src/main/kotlin/ai/digital/integration/server/common/TaskRegistry.kt

```diff
@@ -2,6 +2,10 @@ package ai.digital.integration.server.common
 
```

Comment:
There is no documentation.

Please add this new task and create a new blog how to use it, same as this one https://xebialabs.github.io/integration-server-gradle-plugin/blog/2021/09/02/run-integration-test